pkg/envtest/crd.go

```diff
@@ -225,8 +225,18 @@ func CreateCRDs(config *rest.Config, crds []*apiextensionsv1beta1.CustomResource
 	// Create each CRD
 	for _, crd := range crds {
 		log.V(1).Info("installing CRD", "crd", crd.Name)
-		if _, err := cs.ApiextensionsV1beta1().CustomResourceDefinitions().Create(crd); err != nil {
-			return err
+		if existingCrd, err := cs.ApiextensionsV1beta1().CustomResourceDefinitions().Get(crd.Name, metav1.GetOptions{}); err != nil {
+			if apierrors.IsNotFound(err) {
+				if _, err := cs.ApiextensionsV1beta1().CustomResourceDefinitions().Create(crd); err != nil {
+					return err
+				}
+			}
```

Comment:
should we have a `return err` here?

Reply:
i.e. `if there was an error fetching the existing CRD, and the error wasn't not found, treat it as an error`

Reply:
You're right. Fixing.

Reply:
Done.